awareness.test.js

```diff
@@ -51,3 +52,7 @@ export const testAwareness = tc => {
   t.compare(aw1.getStates().get(0), undefined)
   t.compare(lastChangeLocal, lastChange)
 }
+
+export const testAwareness = testAwarenessWithEncoding()
```

Comment:
This test is the same as the third test and can be removed.

Reply:
I'll augment this test a bit, I'd like to add a test to ensure the default behaviour is using the default encoder.